contrib/python-referencing/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "python-referencing"
+pkgver = "0.32.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
+    "python-build",
+    "python-hatch_vcs",
+    "python-hatchling",
+    "python-installer",
+    "python-rpds",
```

Comment:
this doesn't need to be in hostdeps, you can pass --skip-dependency-check in make_build_args if it's build that complains